homeassistant/components/canary/camera.py

```diff
@@ -3,17 +3,26 @@
 
 import asyncio
 from datetime import timedelta
+from typing import Final
 
+from aiohttp.web_request import Request
+from aiohttp.web_response import StreamResponse
```

Comment:
```suggestion
from aiohttp.web import Request, StreamResponse
```

Reply:
do not trust the auto import completion :see_no_evil: